### PR TITLE
[Translation] Japanese

### PR DESCRIPTION
**Pull request by blauerberg**

### CODE REVIEW

File: config/translations/ja/common.yml [hunk 1]
```diff
@@ -1,44 +1,46 @@
...
-    module: 'Enter the module name'
-    confirm: 'Do you confirm generation?'
-    canceled: 'Command generation canceled.'
+    module: モジュール名を入力してください
+    confirm: 本当に生成しますか？
+    canceled: 生成をキャンセルしました
     events:
-        message: "\nType the event name or use keyup or keydown.\nThis is optional, press <info>enter</info> to <info>continue</info>\n"
-        name: 'Enter event name'
+        message: "\nサービス名を入力するか、方向キーの上か下を押してください。\nThis is optional, press <info>enter</info> to <info>continue</info>\n"
```
Comment (enzolutions) [inline config/translations/ja/common.yml]: Please translate the whole line to accept the PR

File: config/translations/ja/common.yml [hunk 2]
```diff
@@ -1,44 +1,46 @@
...
+        name: イベント名を入力してください
     services:
-        confirm: 'Do you want to load services from the container'
-        message: "\nType the service name or use keyup or keydown.\nThis is optional, press <info>enter</info> to <info>continue</info>\n"
-        name: 'Enter your service'
+        confirm: サービスをコンテナからロードしますか？
+        message: "\nサービス名を入力するか、方向キーの上か下を押してください。\nThis is optional, press <info>enter</info> to <info>continue</info>\n"
```
Comment (enzolutions) [inline config/translations/ja/common.yml]: Please translate the whole line to accept the PR

**Comment by blauerberg** [comment]
@enzolutions thanks for comment!
I added new patch to fix that.

**Comment by enzolutions** [comment]
@balagan73 could but your translation is a little old now , you need to merge with master again and resolve conflicts.

**Comment by blauerberg** [comment]
@enzolutions ok, merged.

**Comment by enzolutions** [comment]
Thanks @blauerberg

**Pull request merged**